processing-tests/common/test/kotlin/com/livefront/sealedenum/compilation/equality/Flag.kt

```diff
@@ -0,0 +1,120 @@
+package com.livefront.sealedenum.compilation.equality
```

Comment:
I think placing this into `compliation/usecases` makes a lot of sense for ensuring a particular usage of `@GenSealedEnum` works!

Reply:
Yep, agreed. I had previously placed it there when the plan was to use a new parameter on `GenSealedEnum`, but now it's an impl detail, so it makes sense for it to be in `compilation/usecases`

Reply:
Makes sense!